src/container/srv_target.c

```diff
@@ -1192,6 +1199,11 @@ cont_child_destroy_one(void *vin)
 
 		cont_child_stop(cont);
 
+		/*
+		 * This might be racy, as dtx_resync() might yield after
+		 * ds_cont_child_lookup(), but before @sc_dtx_resyncing set,
+		 * we use @sc_fini_cond to guarantee all users exit properly.
+		 */
```

Comment:
@Nasf-Fan I think DTX will check sc_stopping, it call ds_cont_child_lookup() internally and inside ds_cont_child_lookup() it will check sc_stopping and return error if sc_stopping has been set.